src/locales/es.json

```diff
@@ -105,15 +109,21 @@
   "Map all fields": "Map all fields",
   "Mapping name": "Mapping name",
   "Mismatch": "Inconsistencia",
+  "No": "No",
```

Comment:
This entry is missing in en.json. We can remove this if it's not used anywhere or else can add in en.json too.